pkg/graveler/graveler.go

```diff
@@ -1291,7 +1291,7 @@ func (g *Graveler) UpdateBranch(ctx context.Context, repository *RepositoryRecor
 // monitorRetries reports the number of retries of operation while updating
 // branchID of repository.
 func (g *Graveler) monitorRetries(ctx context.Context, tries int, repositoryID RepositoryID, branchID BranchID, operation string) {
```

Comment:
I think that we should also change the variable names passed to this method, i.e. change from `retries` to `tries`, or change the parameter name to retries and pass the correct value without modifying the argument value.